pages/eap/index.md

```diff
@@ -37,6 +37,36 @@ check [our instructions on how to configure your build to support this version](
         <th>Build highlights</th>
         <th>kotlinx library versions</th>
     </tr>
+    <tr>
+        <td><strong>1.4-RC</strong>
+            <p> Released: <strong>July 27, 2020</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v1.4-RC" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+            <ul>
+                <li><a href="https://blog.jetbrains.com/kotlin/2020/07/kotlin-1-4-m3-generating-default-methods-in-interfaces/" target="_blank">New modes for generating default methods</a></li>
+                <li>New functionality to conveniently debug coroutines</li>
+                <li>Improved <code>*.gradle.kts</code> IDE support</li>
+                <li>All source sets now include the standard library dependency by default</li>
+                <li>Simplified management of CocoaPods dependencies</li>
+                <li>Improved Kotlin/JS integrations</li>
+            </ul>
+            <p> For more details, please refer to <a href="http://blog.jetbrains.com/kotlin/2020/07/kotlin-1-4-rc-released/" target="_blank">the blog post</a>.</p>
+         </td>
+        <td>
+            <ul>
+                <li><a href="https://github.com/Kotlin/kotlinx.serialization" target="_blank"><strong>kotlinx.serialization</strong>
+                </a> version: <a href="https://bintray.com/kotlin/kotlinx/kotlinx.serialization.runtime/1.0-M1-1.4.0-rc`" target="_blank">1.0-M1-1.4.0-rc</a></li>
```

Comment:
I believe there is an extra '`' at the end of the bintray link — that breaks it. :)